ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1020,12 +1020,12 @@ sent = Sent timestamp
 sent_timestamp = Sent timestamp
 settings.backup.action = Download current settings
 settings.backup.description = Download a copy of the current settings.
-settings.backup.title = Backup current settings
-settings.backuprestore = Backup/Restore
-settings.backuprestore.description = Here you can download or upload the application settings in JSON format.
-settings.restore.action = Upload new settings
+settings.backup.title = Backup application code
+settings.backuprestore = Backup app code
```

Comment:
used "application code" everwhere else but "app code" here. Unsure if there is a reason for the inconsistency?

Reply:
That's used in the nav sidebar so it's abbreviated so it doesn't line wrap.